src/test/java/com/shaft/api/testPostRequest.java

```diff
@@ -0,0 +1,45 @@
+package com.shaft.api;
+
+import com.shaft.api.RestActions;
+import com.shaft.driver.SHAFT;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class testPostRequest {
+    SHAFT.API api = new SHAFT.API("http://localhost:5000");
+    String body = """
+            {
+              "title": "New Course 1",
+              "description": "Full description of the new course in Arabic"
+            }
+            """;
+    List<List<Object>> parameters = Arrays.asList(Arrays.asList("access_key", "your-access-key"));
+    @Test
```

Comment:
Kindly comment this line to prevent the test from running in the pipeline.

Reply:
Done

Reply:
Something is wrong my friend. Can you check the changed files tab? I can only see the test file.